doc/languages-frameworks/java.section.md

```diff
@@ -36,7 +36,23 @@ Private JARs should be installed in a location like
 `$out/share/package-name`.
 
 If your Java package provides a program, you need to generate a wrapper
-script to run it using a JRE. You can use `makeWrapper` for this:
+script to run it using a JRE. A function named `wrapJavaJar` can be used
+to do this automatically. Any attributes given will be passed to
+`mkDerivation`. It also accepts the following optional attributes:
```

Comment:
```suggestion
`mkDerivation`.

It accepts the following optional attributes:
```

A bit more whitespace helps with skimming and reading.

NB: I would prefer wrapping lines at sentence boundaries, as that helps with restructuring and makes diffs easier on the eye.